project/types.go

```diff
@@ -237,6 +237,7 @@ type Service interface {
 	DependentServices() []ServiceRelationship
 	Containers() ([]Container, error)
 	Scale(count int) error
+	SpecificiesHostPort() bool
```

Comment:
I don't feel too strongly about it either way. It just seemed like something that might be useful.